app/main/views/g_cloud.py

```diff
@@ -526,6 +531,8 @@ def view_project(framework_family, project_id):
 
     if project['outcome']:
         current_project_stage = project['outcome']['result']
+    elif project['readyToAssessAt']:
+        current_project_stage = 'ready_to_assess'
```

Comment:
Need to check this with @DilwoarH

Reply:
If these are strings that define some set of finite states, it would be nice to use a proper enum (e.g. https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/ac6532f32d4141fd830e92c89253251d8ca2e79b/app/main/helpers/search_save_helpers.py#L15)